src/ddmd/errors.d

```diff
@@ -94,33 +94,36 @@ extern (C++) void deprecationSupplemental(const ref Loc loc, const(char)* format
 // Just print, doesn't care about gagging
 extern (C++) void verrorPrint(const ref Loc loc, COLOR headerColor, const(char)* header, const(char)* format, va_list ap, const(char)* p1 = null, const(char)* p2 = null)
 {
+    Console* con = cast(Console*)global.console;
     const p = loc.toChars();
-    if (global.params.color)
-        setConsoleColorBright(true);
+    if (con)
+        con.setColorBright(true);
```

Comment:
The null object pattern comes to mind - you define an object that accepts the calls but does nothing. That way you don't need to test the pointer with every use.